src/EditorFeatures/Core/Implementation/SmartIndent/SmartIndent.cs

```diff
@@ -36,7 +36,7 @@ public void Dispose()
         {
             if (lineToBeIndented == null)
             {
-                throw new ArgumentNullException(@"line");
```

Comment:
Public API calling this method has a parameter named “line”, so original string seems correct.

Reply:
Oh, that's right. I missed that.